atlp-rwanda/e-commerce-lydia-32-bn
# Configuring docker app

**What does this PR do?**

1.add configured dockerfile

**how should this be manually tested?**
1. Install Docker Desktop if you haven't already
2. login into docker-desktop
3. login into terminal 
run `docker login`
4. Pull the Docker image from the registry:
   `docker pull rwigara/team-lydia:demo`
5. run `docker images` to see images available on your local 
6. once you find it  Run the Docker container:
   `docker-compose up`
7. Access the backend application at http://localhost:5000
8. 
**Any background context you want to provide?**
N/A
**Screenshots (if appropriate)**
N/A

  **Feel free to ask if any inconvenience**
